File: 3515-find-if-digit-game-can-be-won/find-if-digit-game-can-be-won.cs
public class Solution {
    public bool CanAliceWin(int[] nums) {
        int doubles = 0;
        int single = 0;
        foreach (int i in nums) {
            if ( i < 10) {
                single += i;
            }
            else {
                doubles += i;
            }
        }
    return single!=doubles;
    }
}

File: 3515-find-if-digit-game-can-be-won/find-if-digit-game-can-be-won.java
class Solution {
    public boolean canAliceWin(int[] nums) {
        int doubles = 0;
        int single = 0;
        for (int i = 0; i < nums.length; i++) {
            if ( nums[i] < 10) {
                single += nums[i];
            }
            else {
                doubles += nums[i];
            }
        }
    return single!=doubles;
    }
}